protractor-typescript-cucumber/Config/config.ts

```diff
@@ -0,0 +1,41 @@
+import {ProtractorBrowser} from 'protractor';
+/*
+The config folder includes all the configuration files
+This example config file displays the basic protractor-cucumber framework configuration
+ts-node compiler is needed for cucumberjs
+tags option for specific scenarios added
+**/
+export let config = {
+
+    directConnect: true,
+
+    baseUrl: 'http://juliemr.github.io/protractor-demo/',
```

Comment:
I have mentioned it here https://github.com/angular/protractor-cookbook/issues/13 reason to go with the above url, I agree it would be better if we use cookbook's testapp to avoid confusions, using github url is kinda hacky and Travis might not like it when we have travis integrated 😄 , it would be better we build it locally with `localhost:8080` let me just update it!